rest/client/client.go

```diff
@@ -269,7 +269,9 @@ func (c *communicatorImpl) makeSender(ctx context.Context, td TaskData, opts []L
 			case apimodels.TaskLogPrefix:
 				c.loggerInfo.Task = append(c.loggerInfo.Task, metadata)
 			}
-		case model.BuildloggerLogSender:
+		case model.EvergreenLogSender:
+			sender = newEvergreenLogSender(ctx, c, prefix, td, bufferSize, bufferDuration)
+		default:
```

Comment:
I think maybe we want to omit the changes to this file.

Reply:
why?

Reply:
because we'll do all of the testing by opting in on a per-project/default setting in the application that we turn on in staging, and then change the code later.

Reply:
doesn't the project settings come from the project's yaml config? which we do not have control over unless it is one of our projects

Reply:
or the database/project/ref/global config?

Reply:
not sure what that is